src/Parser/VengeanceDemonHunter/CombatLogParser.js

```diff
@@ -127,7 +128,13 @@ class CombatLogParser extends MainCombatLogParser {
 
     results.statistics = [
       <StatisticBox
-        icon={<Icon icon="class_demonhunter" alt="Damage taken" />}
+        icon={<Icon icon="class_demonhunter" alt="Damage done" />}
+        value={`${formatNumber(this.totalDamageDone / this.fightDuration * 1000)} DPS`}
+        label='Damage done'
```

Comment:
Yes. But as a DH tank, I always like to watch my (overpowered) DPS meters too. e.g. I do aprox. 800-900k dps on Desolate Host, so I think it is a good meter to measure how I'm helping my raid team in the DPS as well.